### PR TITLE
Gough 9-10

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: history-of-the-quakers-v3/original/29-book-6-chapter-9.adoc [hunk 1]
```diff
@@ -157,8 +171,13 @@ William Penn placed himself in the balcony, and thence preached to the people;
 but in the midst of his declaration he was interrupted
 by officers from the mayor with the following warrant:
 
+[.embedded-content-document.legal]
+--
+
+[.signed-section-context-open]
 Wells City and Borough.
```
Comment (maijapapaya) [inline history-of-the-quakers-v3/original/29-book-6-chapter-9.adoc:178]: Wasn't sure if this should be above the [.letter-heading] or not, or if I should move it around or not even label it anything.

File: history-of-the-quakers-v3/original/29-book-6-chapter-9.adoc [hunk 2]
```diff
@@ -200,8 +223,11 @@ London renewed their application to parliament,
 for accepting the solemn affirmation of the members of their society instead of an oath,
 by the following case and petition.
 
-The suffering case of the People commonly called Quakers, relating to oaths and swearing,
-humbly offered.
+[.embedded-content-document.address]
+--
+
+[.blurb]
+=== The suffering case of the People commonly called Quakers, relating to oaths and swearing, humbly offered.
```
Comment (maijapapaya) [inline history-of-the-quakers-v3/original/29-book-6-chapter-9.adoc:230]: I seem to not be able to comment where I had left a note for myself to, but lines 294, 297, and 299 look like another letter but they are a case and a petition within the document they are embedded in. I wasn't sure if I should separate them from it or not into their own [embedded-content-document.].

Comment (jaredh159) [reply history-of-the-quakers-v3/original/29-book-6-chapter-9.adoc:230]: Yes, I think you should separate them into another document. 👍

**Comment by jaredh159** [comment]
Thanks, merged. 👍

**Pull request merged**